src/app/_storage/settings.ts

```diff
@@ -38,6 +38,7 @@ export class MessageSettings {
   scaling = 8;
   offset = 0;
   text = 'Hello world!';
+  texts = [];
```

Comment:
This would probably be better if we had all messages within a "messages" array including the original one. I'm sure you did this for compatibility, in which case on load for a little bit there should be a migration from the old message to the new one. I'm already doing something similar, see [app.component.ts](https://github.com/bluecaret/carettab/blob/07addfb79d00e2db6b4509310bf6a3f3c3683e44/src/app/app.component.ts#L98)